contract/test/agd-lib.test.js

```diff
@@ -0,0 +1,121 @@
+import test from 'ava';
+import { makeAgd, makeContainer } from '../tools/agd-lib.js';
```

Comment:
I'd love to see a JS library that powers both the agd CLI and our needs for agd operations elsewhere

Reply:
I tend to track this as...

 - https://github.com/Agoric/agoric-sdk/issues/8963

Reply:
That's a good one. I have in mind something smaller in scope. I filed,
- https://github.com/Agoric/agoric-sdk/issues/10168